addons/sale_coupon/models/sale_order.py

```diff
@@ -514,6 +515,14 @@ def _compute_tax_id(self):
             taxes = line.tax_id.filtered(lambda r: not line.company_id or r.company_id == line.company_id)
             line.tax_id = fpos.map_tax(taxes, line.product_id, line.order_id.partner_shipping_id) if fpos else taxes
 
+    def _get_display_price(self, product):
+        # A product created from a promotion does not have a list_price.
+        # The price_unit of a reward order line is computed by the promotion, so it can be used directly
+        if self.is_reward_line:
+            return self.price_unit
+        else:
```

Comment:
don't need the else.

`super(SaleOrderLine, self)._get_display_price(product)` -> `super()._get_display_price(product)`